homeassistant/components/mysensors/cover.py

```diff
@@ -132,3 +136,44 @@ async def async_stop_cover(self, **kwargs: Any) -> None:
         self.gateway.set_child_value(
             self.node_id, self.child_id, set_req.V_STOP, 1, ack=1
         )
+
+    @property
+    def current_cover_tilt_position(self) -> int | None:
+        """Return current position of cover tilt."""
+        set_req = self.gateway.const.SetReq
+        if hasattr(set_req, "V_TILT"):
+            return self._values.get(set_req.V_TILT)
+        return None
+
+    async def async_set_cover_tilt_position(self, **kwargs: Any) -> None:
+        """Move the cover tilt to a specific position."""
+        set_req = self.gateway.const.SetReq
+        if hasattr(set_req, "V_TILT"):
```

Comment:
If the device supports tilt, we should update the supported features of the cover entity. We shouldn't have a condition here.

Reply:
I guess we don't know when that message is received. Ok, then let's consolidate the check to `current_cover_tilt_position`.